[2024-06-02T15:11:55.883+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: alert.run_detect_anomalies_script scheduled__2024-06-01T11:00:00+00:00 [queued]>
[2024-06-02T15:11:55.922+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: alert.run_detect_anomalies_script scheduled__2024-06-01T11:00:00+00:00 [queued]>
[2024-06-02T15:11:55.926+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2024-06-02T15:11:56.038+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): run_detect_anomalies_script> on 2024-06-01 11:00:00+00:00
[2024-06-02T15:11:56.070+0000] {standard_task_runner.py:60} INFO - Started process 8301 to run task
[2024-06-02T15:11:56.147+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'alert', 'run_detect_anomalies_script', 'scheduled__2024-06-01T11:00:00+00:00', '--job-id', '699', '--raw', '--subdir', 'DAGS_FOLDER/alert.py', '--cfg-path', '/tmp/tmpg4_aqziw']
[2024-06-02T15:11:56.167+0000] {standard_task_runner.py:88} INFO - Job 699: Subtask run_detect_anomalies_script
[2024-06-02T15:11:56.966+0000] {task_command.py:423} INFO - Running <TaskInstance: alert.run_detect_anomalies_script scheduled__2024-06-01T11:00:00+00:00 [running]> on host 6f8b2a20b5b0
[2024-06-02T15:11:58.210+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='alert' AIRFLOW_CTX_TASK_ID='run_detect_anomalies_script' AIRFLOW_CTX_EXECUTION_DATE='2024-06-01T11:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-01T11:00:00+00:00'
[2024-06-02T15:11:58.567+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-06-02T15:11:58.642+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=alert, task_id=run_detect_anomalies_script, execution_date=20240601T110000, start_date=20240602T151155, end_date=20240602T151158
[2024-06-02T15:11:58.782+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-06-02T15:11:59.087+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
